pkgs/development/compilers/bs-platform/build-bs-platform.nix

```
@@ -30,9 +30,7 @@ stdenv.mkDerivation {
     cp ${custom-ninja}/bin/ninja vendor/ninja/snapshot/ninja.linux
   '';
 
-  configurePhase = ''
-    node scripts/ninja.js config
-  '';
+  dontConfigure = true;
```

do you mind leaving a comment, so we don't stumble upon this again
```suggestion
  # avoid building the development version, will break aarch64 build
  dontConfigure = true;
```